content/workshops/openshift_service_mesh/lab1.3_deploymsa.md

```diff
@@ -51,7 +51,7 @@ Deploy the boards service:
 ```
 oc new-app -f ./openshift-configuration/boards-fromsource.yaml \
   -p APPLICATION_NAME=boards \
-  -p NODEJS_VERSION_TAG=node-12 \
+  -p NODEJS_VERSION_TAG=node-10 \
```

Comment:
Do I have a different tag setup than you?  I'm worried the buildconfig is pulling a different imagestream.  I've got 'nodejs:10' where the imagestream 'nodejs' has tags '10, 8, 8-RHOAR, latest'.

Reply:
hmm. let me check.

Reply:
yup - my bad, fixing to use the on that available by default in the RHPDS cluster image stream